src/Faker/Calculator/Inn.php

```diff
@@ -11,27 +15,24 @@ class Inn
      *
      * @param string $inn
      * @return string Checksum (one digit)
+     * @deprecated use {@link \Faker\Provider\ru_RU\Company::inn10Checksum()} instead
+     * @see \Faker\Provider\ru_RU\Company::inn10Checksum()
      */
     public static function checksum($inn)
     {
-        $multipliers = [2, 4, 10, 3, 5, 9, 4, 6, 8];
-        $sum = 0;
-
-        for ($i = 0; $i < 9; $i++) {
-            $sum += (int) $inn[$i] * $multipliers[$i];
-        }
-
-        return (string) (($sum % 11) % 10);
+        return \Faker\Provider\ru_RU\Company::inn10Checksum($inn);
     }
 
     /**
      * Checks whether an INN has a valid checksum
      *
      * @param string $inn
      * @return bool
+     * @deprecated use {@link \Faker\Provider\ru_RU\Company::inn10IsValid()} instead
+     * @see \Faker\Provider\ru_RU\Company::inn10IsValid()
      */
     public static function isValid($inn)
     {
-        return strlen($inn) === 10 && self::checksum($inn) === $inn[9];
+        return \Faker\Provider\ru_RU\Company::inn10IsValid($inn);
```

Comment:
Same here

Reply:
@pimjansen the problem here is that `INN` and `TCNo` _are_ locale specific but somehow ended up in the base `Calculator`. 

We keep them in `Calculator` (and call the locale specific providers) and mark them deprecated for BC , but they should be removed from the `Calculator` asap IMO - especially considering Faker 2.0, where it makes no sense to have this in the core.